spec.html

```diff
@@ -7121,7 +7121,7 @@ <h1>
 
     <emu-clause id="sec-createiterresultobject" type="abstract operation">
       <h1>
-        CreateIterResultObject (
+        CreateIteratorResultObject (
```

Comment:
Maybe worth giving the clause an ID to match the new name? With an oldid, of course.